### PR TITLE
Load file + tests & fixes

### PR DESCRIPTION
**Pull request by erikh**
Closes #15 

This extends #15 and adds some tests and errno checking.

### CODE REVIEW

File: mruby.go [hunk 1]
```diff
@@ -176,6 +179,34 @@ func (m *Mrb) LoadString(code string) (*MrbValue, error) {
 	return newValue(m.state, value), nil
 }
 
+// LoadFile loads the given file, executes it, and returns its final
+// value that it might return.
+func (m *Mrb) LoadFile(file string) (*MrbValue, error) {
+
```
Comment (mitchellh) [inline mruby.go]: Superflous newline

**Comment by mitchellh** [comment]
One nitpick, but thanks for adding the test

**Comment by erikh** [comment]
fixed

**Comment by erikh** [comment]
I'm going to merge this if that's OK. Let me know if you object and I will revert/re-patch.

**Pull request merged**